<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_31">
      <item row="2" column="0">
       <widget class="QDockWidget" name="spectrometer_control_dockWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle">
         <string>Spectrometer</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_cooler_on">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Cooler on</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_cooler_off">
              <property name="text">
               <string>Cooler off</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2" rowspan="12">
           <widget class="PlotWidget" name="spectrum_PlotWidget"/>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Wavelengths [nm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="set_wavelength_doubleSpinBox">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>400.000000000000000</double>
              </property>
              <property name="maximum">
               <double>800.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="set_temperature_pushButton">
              <property name="text">
               <string>Set Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="set_temperature_doubleSpinBox">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
              <property name="maximum">
               <double>30.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Number of accumulations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="number_of_accumulations_doubleSpinBox">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Exposure time [s]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="exposure_time_doubleSpinBox">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Cycle time [s]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="cycle_time_doubleSpinBox">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="take_spectrum_pushButton">
              <property name="text">
               <string>Take Spectrum</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="continuous_spectrum_acquisition_radioButton">
              <property name="text">
               <string>Continuous</string>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="take_background_pushButton">
              <property name="text">
               <string>Take Background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reset_background_pushButton">
              <property name="text">
               <string>Reset Background</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Grating</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="grating_spinBox">
              <property name="maximum">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_31">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>xmin [nm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spectrum_xmin_spinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_32">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>xmax [nm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spectrum_xmax_spinBox">
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="10" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_33">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>ymin [nm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spectrum_ymin_spinBox">
              <property name="minimum">
               <number>-10000000</number>
              </property>
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="10" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_34">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>ymax [nm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spectrum_ymax_spinBox">
              <property name="maximum">
               <number>10000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="0">
           <widget class="QRadioButton" name="spectrum_autorange_radioButton">
            <property name="text">
             <string>Autorange</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="spectrum_save_data_pushButton">
            <property name="text">
             <string>Save Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
